src/main/java/io/github/nuclearfarts/cbt/mixin/ReloadableResourceManagerImplMixin.java

```diff
@@ -24,8 +24,9 @@
 	@Shadow
 	public abstract void addPack(ResourcePack resourcePack);
 
-	@Inject(method = "reload", at = @At(value = "RETURN", shift = At.Shift.BEFORE))
+	@Inject(method = "reload", at = @At(value = "INVOKE",target = "Ljava/util/List;iterator()Ljava/util/Iterator;", shift = At.Shift.AFTER))
 	private void injectCBTPack(Executor prepareExecutor, Executor applyExecutor, CompletableFuture<Unit> initialStage, List<ResourcePack> packs, CallbackInfoReturnable<ResourceReload> cir) {
+		System.out.println("reload");

```

Comment:
Could you remove the System.out.println call you added here that seems to have been left in from debugging?